lib/ret_web/controllers/api/v1/oauth_controller.ex

```diff
@@ -1,24 +1,30 @@
 defmodule RetWeb.Api.V1.OAuthController do
   use RetWeb, :controller
 
-  alias Ret.{Repo, OAuthToken, OAuthProvider, DiscordClient, TwitterClient, Hub, Account, PermsToken}
+  alias Ret.{Repo, OAuthToken, OAuthProvider, DiscordClient, SlackClient, TwitterClient, Hub, Account, PermsToken}
   import Canada, only: [can?: 2]
 
   plug(RetWeb.Plugs.RateLimit when action in [:show])
 
-  def show(conn, %{"type" => "discord", "state" => state, "code" => code}) do
-    %{claims: %{"hub_sid" => hub_sid}} = OAuthToken.peek(state)
+  # handle twitter oauth to allow users hubs to tweet
```

Comment:
nit: Generally, I think comments like these are not particularly useful, since you can just read the code and see what it's doing. This comment might also not be true in the future, if we decide to use twitter oauth for other purposes.